include/evmc/utils.h

```diff
@@ -10,3 +10,9 @@
 #else
 #define EVMC_EXPORT __attribute__((visibility("default")))
 #endif
+
+#if __cplusplus
+#define EVMC_NOEXCEPT noexcept
```

Comment:
I was thinking about naming it just `NOEXCEPT` but checking if the macro was not previously defined.

Reply:
The same could be said for `EXPORT` I think? A lot of systems use it for the same purpose.

Reply:
I would go for it.